src/main/scala/org/deepdive/inference/SQLInferenceRunner.scala

```diff
@@ -547,11 +547,7 @@ trait SQLInferenceRunner extends InferenceRunner with Logging {
                 |= t1.${dataStore.quoteColumn(s"${InferenceNamespace.getBaseTableName(v.relation)}.id")}
                 |""".stripMargin.replaceAll("\n", " ")).mkString("AND")
             val weightJoinlist = factorDesc.weight.variables.map(v => {
-              // split column to get relation name
-              val colSplit = v.split('.')
-              val lastvrel = InferenceNamespace.getBaseTableName(colSplit(0))
-              val lastv = s"${lastvrel}.${colSplit.takeRight(colSplit.length-1).mkString(".")}"
-              s""" t0.${dataStore.quoteColumn(v)} = t1.${dataStore.quoteColumn(lastv)}"""
+              s""" t0.${dataStore.quoteColumn(v)} = t1.${dataStore.quoteColumn(v)}"""
```

Comment:
Is this backwards compatible?  Why did we need such complicated column name manipulation before if the same name could be just shared across t0 and t1?  Does this require any change to t0 or t1?